### PR TITLE
[WIP] Updated doco to mention 4.5.2

### PR DESCRIPTION
**Pull request by johnsimons**
Merge only once we have deployed v6 and PI has been updated

### CODE REVIEW

File: platform/installer/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The Platform Installer is a Microsoft Click-Once application which means it has
 
 ### Dependencies
 
-The Click-Once setup.exe will install [.Net 4.5](http://www.microsoft.com/en-au/download/details.aspx?id=40779) if required and with then bootstrap the Platform Installer Application
+The Click-Once setup.exe will install [.Net 4.5.2](https://www.microsoft.com/en-us/download/details.aspx?id=42643) if required and with then bootstrap the Platform Installer Application
```
Comment (mikeminutillo) [inline platform/installer/index.md]: > and *will* then bootstrap

File: platform/installer/offline.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The [Platform Installer](/platform/installer) handles installing prerequisites f
 
 ## .Net Prerequisite Version
 
-The Particular platform products require [.Net 4.5](https://www.microsoft.com/en-au/download/details.aspx?id=30653). Before proceeding please ensure that this version of .Net is available on the system you are setting up. More detailed information about .NET can be found on the [Wikipedia .NET Overview](https://en.wikipedia.org/wiki/.NET_Framework_version_history#Overview).
+The Particular platform products require [.Net 4.5.2](https://www.microsoft.com/en-us/download/details.aspx?id=42643). Before proceeding please ensure that this version of .Net is available on the system you are setting up. More detailed information about .NET can be found on the [Wikipedia .NET Overview](https://en.wikipedia.org/wiki/.NET_Framework_version_history#Overview).
```
Comment (mikeminutillo) [inline platform/installer/offline.md]: Is it .Net or .NET?

The two downloads used to be for different Ids. Was there a good reason for that?

Comment (SimonCropp) [inline platform/installer/offline.md]: `.NET`

**Comment by SimonCropp** [comment]
merged manually here https://github.com/Particular/docs.particular.net/commit/d7c5fa6b64a50f0ab9ebe3e62138ad458e3f04de

**Pull request closed without merging**